src/workerd/io/worker-fs.c++

```diff
@@ -13,6 +13,36 @@ namespace workerd {
 KNOWN_VFS_ROOTS(DEFINE_DEFAULTS_FOR_ROOTS)
 #undef DEFINE_DEFAULTS_FOR_ROOTS
 
+static kj::Path tmpPath = kj::Path::parse("tmp");
+
+// Helper function to get the current working directory path.
+// Returns the Cwd if available, otherwise returns an empty path (root).
+kj::PathPtr getCurrentWorkingDirectory() {
+  if (IoContext::hasCurrent()) {
+    return IoContext::current().getTmpDirStoreScope().getCwd();
+  }
+  if (TmpDirStoreScope::hasCurrent()) {
+    return TmpDirStoreScope::current().getCwd();
+  }
+  // Default to "bundle" if no context is available
+  // this way bundle is used during init
```

Comment:
```suggestion
  // Default to "tmp" if no context is available
  // this way bundle is used during init
```

Reply:
Ahh this was a type in the implementation not the comment - should have been `"bundle"`.